common/temp_modules/npmx-gulp-core-build-karma/package.json

```diff
@@ -29,6 +29,6 @@
     "webpack": "~1.13.0"
   },
   "npmxDependencies": {
-    "@microsoft/gulp-core-build": ">=1.0.1 <2.0.0"

```

Comment:
Under the current design of NPMX, they must be in Git. I agree that the name "temp" implies the opposite.

Originally I intended for them to be temporary, since the same information already exists in your package.json file.  However, if you changed package.json and forgot to run "npmx generate", then the current algorithm would have a hard time detecting/explaining the problem. That's why they are tracked by Git.

We discussed making "npmx generate" smarter and merging it with "npmx install".  If we do that, then these temp_modules could really be temporarily.